src/libmongoc/tests/test-mongoc-shared.c

```diff
@@ -0,0 +1,86 @@
+
+
+#include "./TestSuite.h"
```

Comment:
Minor, but for consistency with other test includes:

```suggestion
#include "TestSuite.h"
```